### PR TITLE
[taxii2-server] - fix bug when creating relationships

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Hold - (Reason for hold)
- [ ] In Progress
- [x] Ready

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-8805)

## Description
Fixed an issue where when trying to create indicator relationships the code failed on `AttributeError`

### CODE REVIEW

**Review by MLainer1 — APPROVED** [review]
😎

**Review by Ni-Knight — APPROVED** [review]

File: Packs/TAXIIServer/ReleaseNotes/2_0_50.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### TAXII2 Server
+
+- Fixed an issue where the integration failed to create indicator relationships in some cases.
```
[the remark this answers is not in the document]
Comment (Ni-Knight) [reply Packs/TAXIIServer/ReleaseNotes/2_0_50.md:6]: @GuyAfik - Need a bit more clarification in the comment.

[the remark this answers is not in the document]
Comment (GuyAfik) [reply Packs/TAXIIServer/ReleaseNotes/2_0_50.md:6]: I will add

**Comment by GuyAfik** [comment]
requires force-merge as the taxii2 performance playbook does not pass which is un-related to the changes in this PR, in addition opened an issue about

**Pull request merged**